docs/how-to/create-beamline.rst

```diff
@@ -43,13 +42,27 @@ The following example creates a fictitious beamline ``w41``, with a simulated tw
     to one or more Bluesky Protocols.
     """
 
+    """
+    A valid factory function which:
+    - may be instantiated automatically, selectively on live beamline
+        - a maximum of once
+    - automatically names the device
+    - must be explicitly connected (may be automated by tools)
+        - when connected may connect to a simulated backend
+        - may be connected concurrently (when automated by tools)
+    """"
+    @device_factory(skip=BL == "s41")
+    def synchrotron() -> Synchrotron:
+        return Synchrotron()
+
 
     """
-    A valid factory function which is:
-    - instantiated only on the live beamline
-    - a maximum of once
-    - can optionally be faked with ophyd simulated axes
-    - can optionally be connected concurrently by not waiting for connect to complete
+    A valid factory function which:
+    - may be instantiated automatically, selectively on live beamline
+        - a maximum of once
+    - must attempt to connect when first called
+        - can optionally be faked with ophyd simulated axes
+        - can optionally be connected concurrently by not waiting for connect to complete
     - if constructor took a prefix, could optionally exclude the BLIXX prefix
     """"
     @skip_device(lambda: BL == "s41")  # Conditionally do not instantiate this device
```

Comment:
Should: This is the old way of doing things. I think it's fine that the behavior is still there whilst we're depreciating it but we shouldn't be documenting it in a how to otherwise people might use it for new stuff